upload.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+
+# usage: ./upload.sh mapbox-username
+# set a MapboxAccessToken environment variable before running, e.g.
+# export MapboxAccessToken=<access token with uploads:write scope enabled>
+# ./upload.sh rodowi
+
+npm install --global mapbox-upload
```

Comment:
This wouldn't work on a typical Linux machine (npm install -g requires su).

Can't we have mapbox-upload as a depencency in [package.json](https://github.com/hotosm/osm-analytics-cruncher/blob/master/package.json#L28) and use  the locally installed binary (like [here](https://github.com/hotosm/osm-analytics-cruncher/blob/master/hotprojects.sh#L4))?